### ISSUE TITLE
Design a logo for fact-bounty

### ISSUE THREAD

**Issue by root5533** [scorelab/fact-Bounty#93]
Currently we don't have an official logo for fact-bounty. The logo image should be in SVG format(simple, minimalistic). If anyone likes to contribute to this, please comment your designs in this issue.

**Comment by kartikpandey2** [comment]
@root5533 

![Fact-Bounty Logo](https://user-images.githubusercontent.com/20434963/53721395-2ee96480-3e89-11e9-99f2-2ce0d8566910.png)

**Comment by root5533** [comment]
Doesn't look really attractive. Let's try something simpler but modern

**Comment by thivatm** [comment]
![01artboard 1 4x](https://user-images.githubusercontent.com/25591638/53747590-9f17da80-3ec9-11e9-9ff3-62dc2ab13fef.png)
How about this? @root5533

**Comment by kartikpandey2** [comment]
checkout this @root5533 

![Fact-Bounty Logo](https://user-images.githubusercontent.com/20434963/53800433-b7880380-3f62-11e9-8491-31a7968cad33.png)

**Comment by thobhanifreddy** [comment]
![asset 1](https://user-images.githubusercontent.com/12369649/53824366-cc33be00-3f99-11e9-9abb-0cd14cb1a23c.png)

**Comment by kartikpandey2** [comment]
@root5533 @ivantha  Is it good enough to be our logo .? 😅 
![logo](https://user-images.githubusercontent.com/20434963/54646679-e60af000-4ac5-11e9-9f91-e74139862984.png)

**Comment by wdevon99** [comment]
![fb-logo](https://user-images.githubusercontent.com/28486947/54771021-74988200-4c2a-11e9-9b75-1599b46e8e10.jpg)

Just made a simple logo! If more requirements are can be specified, I can make a logo accordingly :)

**Comment by wdevon99** [comment]
![logo](https://user-images.githubusercontent.com/28486947/60231934-9a33c800-98b8-11e9-93e6-a3752f8aef07.png)

Another attempt to make a logo :p

**Comment by root5533** [comment]
Let's at the moment keep the FactBounty blue yellow theme and the word. But the logo is not quite ok and having it with word FB is not quite ok in my opinion. Color theme is fine for now

**Comment by wdevon99** [comment]
> Let's at the moment keep the FactBounty blue yellow theme and the word. But the logo is not quite ok and having it with word FB is not quite ok in my opinion. Color theme is fine for now

Yeah, FB sound more like facebook, I agree, ill try to come up with something else :) @root5533

**Comment by kunalbhatia18** [comment]
![fact](https://user-images.githubusercontent.com/39365087/70201962-1cfd6080-173e-11ea-98a6-b1d4bd02de48.png)
Hope you like it :)

**Comment by kunalbhatia18** [comment]
![fb](https://user-images.githubusercontent.com/39365087/70202007-3bfbf280-173e-11ea-8445-9e4f2e4628e1.png)
Almost design as previous ones but just to have a variety...this might be good too

**Comment by wdevon99** [comment]
> ![fact](https://user-images.githubusercontent.com/39365087/70201962-1cfd6080-173e-11ea-98a6-b1d4bd02de48.png)
> ![ffb](https://user-images.githubusercontent.com/39365087/70202297-48cd1600-173f-11ea-8e64-e56453b5e33f.png)
> 
> Hope you like it :)

Looks good! It's good if we can have the full name in the logo rather than FB.

**Comment by kunalbhatia18** [comment]
> > ![fact](https://user-images.githubusercontent.com/39365087/70201962-1cfd6080-173e-11ea-98a6-b1d4bd02de48.png)
> > ![ffb](https://user-images.githubusercontent.com/39365087/70202297-48cd1600-173f-11ea-8e64-e56453b5e33f.png)
> > Hope you like it :)
> 
> Looks good! It's good if we can have the full name in the logo rather than FB.

Sure lemme share another one with a similar design

**Comment by kunalbhatia18** [comment]
![fb](https://user-images.githubusercontent.com/39365087/70217727-bbea8280-1767-11ea-92e7-56423c98bbdc.png)
![fbb](https://user-images.githubusercontent.com/39365087/70217736-c278fa00-1767-11ea-8ac4-e0ccdfdf27fa.png)
How about this?

**Comment by wdevon99** [comment]
@kunalbhatia18 much better :)

**Comment by SufiyanPasta** [comment]
![5dfcf6d0-b53b-40e2-a530-b7f5ec2ac27f_200x200](https://user-images.githubusercontent.com/58876317/70846217-e8458380-1e78-11ea-8dbd-f6624112abc0.png)

How about this? It should reflect what Fact Bounty Do

**Comment by bunny8469** [comment]
This is my work.
Hope you like it.
![Fact Bounty-logos](https://user-images.githubusercontent.com/58695175/71434893-7ee13280-270c-11ea-80a1-4582a2b256f1.png)

**Comment by JosephSemrai** [comment]
After looking at the comments above, I think this might fit most of the criteria:
![image](https://user-images.githubusercontent.com/29003194/71461224-18e0c380-277d-11ea-89d8-fb2e65567760.png)

**Comment by wdevon99** [comment]
> After looking at the comments above, I think this might fit most of the criteria:
> ![image](https://user-images.githubusercontent.com/29003194/71461224-18e0c380-277d-11ea-89d8-fb2e65567760.png)
> 
> SVG File: https://drive.google.com/file/d/1M3Gr2fH62wLIB2uIReQj87NhBssTKWgP/view?usp=sharing

Good Job!

**Comment by jakubpawlina** [comment]
![FactBounty](https://user-images.githubusercontent.com/29257057/71592368-412b4200-2b30-11ea-8654-ffe0634dd999.png)
Maybe something like this?
Download [svg](https://bit.ly/2QaWOmH) file.

**Comment by jakubpawlina** [comment]
![g1334](https://user-images.githubusercontent.com/29257057/71593780-a2a1df80-2b35-11ea-8f37-c3291642e03a.png)
Some other color options, on dark background.

**Comment by sharman13** [comment]
@kunalbhatia18 Can you tell me how many tasks you have completed in google code in?

**Comment by kunalbhatia18** [comment]
> @kunalbhatia18 Can you tell me how many tasks you have completed in google code in?
Ya sure! Can you DM me on Gitter?

**Comment by SharmaIND** [comment]
![20200101_190427_0000](https://user-images.githubusercontent.com/59284021/71799483-68779700-307b-11ea-85df-bc2f3fc19a32.png)

**Comment by cjprogamer** [comment]
![fb](https://user-images.githubusercontent.com/41909955/72947253-70d40f80-3da7-11ea-8721-60fa7dcec1b6.png)
How about this logo ?

**Comment by ghost** [comment]
![14fc3442-c450-4b29-9009-940eb3eb1a73_200x200 svg](https://user-images.githubusercontent.com/58349712/86527174-0b3d9080-beba-11ea-9f8f-cf7b1528956c.png)
